java/src/main/java/com/powsybl/python/PyPowsyblApiLib.java

```diff
@@ -916,8 +933,8 @@ private static String getBusId(Terminal t) {
                             .addDoubleSeries("max_p", HvdcLine::getMaxP)
                             .addDoubleSeries("nominal_v", HvdcLine::getNominalV)
                             .addDoubleSeries("r", HvdcLine::getR)
-                            .addStringSeries("converter_station1", l -> l.getConverterStation1().getId())
-                            .addStringSeries("converter_station2", l -> l.getConverterStation2().getId()))
+                            .addStringSeries("converter_station1_id", l -> l.getConverterStation1().getId())
+                            .addStringSeries("converter_station2_id", l -> l.getConverterStation2().getId()))
```

Comment:
I guess we should add voltage levels here too ?

Reply:
Not needed because we can join with converter stations which have a voltage level id

Reply:
ok